OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -368,8 +368,8 @@ function Set-TargetResource {
     $stopRequested = $State -eq "Stopped"
     $startRequested = $State -eq "Started"
     $removeRequested = $Ensure -eq "Absent"
-    $installRequested = $Ensure -eq "Install"
```

Comment:
Fixed a bug where it was trying to configure it, even though I was asking it to install msi only